### ISSUE TITLE
[Virtual On] Freezes after winning

### ISSUE THREAD

**Issue by devmiyax** [devmiyax/yabause#532]
Master SH2 invalid opcode

R0 =  00000130	R12 =  00000000
R1 =  06032ADC	R13 =  00000001
R2 =  00002200	R14 =  0603A000
R3 =  06034C28	R15 =  06001FAC
R4 =  06032ADC	SR =   00000001
R5 =  8B0C85E2	GBR =  00000000
R6 =  61E0611C	VBR =  06000000
R7 =  060D59EC	MACH = 00000000
R8 =  06035200	MACL = 00000000
R9 =  00000A80	PR =   06018AB8
R10 = 00000500	PC =   00002200
R11 = 0601618C

**Comment by devmiyax** [comment]
0x06032AD0 0002200

**Comment by devmiyax** [comment]
0x00002200: unrecognized
0x06018AB6: mov r1, r4
0x06018AB4: jsr @r2
0x06018AB2: mov.l @(0x004, r1), r2
0x06018AB0: mov.l r1, @r15
0x06018AAE: mov.l @(r0, r14), r1
0x06018AAC: mov.w @(0x09C, pc), r0 ; 0x06018B48
0x06018AAA: mov.l r2, @r3
0x06018AA8: mov.l @(0x0BC, pc), r3 ; 0x06018B64
0x06018AA6: mov #0x00, r2

**Comment by devmiyax** [comment]
Read from CS2 io port

```
R00 = 00000004
R03 = 00002200
R04 = 25818000
R05 = 06032ADC

0x06016226: mov.l r3, @(r0, r5)
0x06016224: mov.l @r4, r3
0x06016222: shll2 r0
0x06016220: mov.l @r15, r0
```

**Comment by devmiyax** [comment]
cs2	: Command: playDisc 0fd4 10ff ffff ffff ffff
cs2	: playDisc: Unsupported play mode = 7F
Sector filtered out
Sector filtered out
Sector filtered out
Sector filtered out
Sector filtered out
Sector filtered out

**Comment by devmiyax** [comment]
Fixed
![image](https://user-images.githubusercontent.com/1174152/39096440-25d0cc18-468b-11e8-9b4d-e30378e8ccb7.png)

**Comment by devmiyax** [comment]
![image](https://user-images.githubusercontent.com/1174152/39096479-a05a3348-468b-11e8-8e70-d3999bc172bc.png)

**Comment by fivefeet8** [comment]
Fixed in 1.8
